rfcs/0015-release-manager.md

```diff
@@ -0,0 +1,79 @@
+---
+feature: release-manager-nixos
+start-date: 2017-07-18
+author: Robin Gloster (@globin)
+co-authors: Franz Pletz (@fpletz)
+related-issues: --
+---
+
+# Summary
+[summary]: #summary
+
+NixOS currently has no process for electing release managers (RMs). We propose to
+switch to a model with two RMs, where each RM SHOULD
+serve for a consecutive term of two releases. A new RM is appointed
+by the previous team for each new release.
+
+# Motivation
+[motivation]: #motivation
+
+Currently release managing in NixOS has mostly been done by individuals who
+volunteered and were then chosen by the last release manager. Over the last
+few releases a process has been established and
+[documented](https://nixos.org/nixos/manual/index.html#release-process).
+As this makes it easier to cut a release this role should be passed on
+regularly and not be held by a single individual over a longer time.
```

Comment:
What problem does it solve? Is it hard to find a release manager,  is there too much demand, ...